docs/01-app/01-getting-started/06-data-fetching-and-streaming.mdx

```diff
@@ -87,6 +89,13 @@ export default async function Page() {
 
 ### Client Components
 
+There are two ways to fetch data in Client Components, using:
```

Comment:
I mean, i can also just use fetch in a `useEffect` right? But maybe we are steering folks away from that pattern...

Reply:
Yes, that's right! But it's not really recommended, so I thought it might be best to omit it in favor of `use`